test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -146,14 +146,19 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        // [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
         [TestCategory("Multiclass")]
         [TestCategory("Logistic Regression")]
-        //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
-        [Trait("Category", "SkipInCI")]
-        public void MulticlassLRTest()
+        [Theory, VaryingTolerance(3)]
```

Comment:
Have you run these tests with IterationData?
For this test and the one below, how often do we see failures if we retain the tolerance at the original level? And if we reduce the tolerance by one as you have done, do the failures disappear completely?

Reply:
These tests passed in this [build ](https://github.com/dotnet/machinelearning/pull/4855/commits/6062f629f49e1ff9b5fd54f373343c5ff452ef4f) with 1000 iterations. Therefore I'm removing the VartingTolerance Theory tag from these tests.